external/report/report/create_report.py

```diff
@@ -99,6 +99,14 @@ def __repr__(self) -> str:
         return "<ol>\n" + "\n".join(items_li) + "\n</ol>"
 
 
+class RawHTML:
```

Comment:
I thought we already had an object that did this. I could be wrong though.

Reply:
You're right, there was one defined in `views/matplotlib.py`. I refactored the functions that used it to use the new `RawHTML` from `report`.